dist/index.html

```diff
@@ -20,7 +20,7 @@ <h2 class="user-info" id="user-info">Welcome, User!</h2>
     <main class="main">
       <section class="filter-section" id="filterSection">
         <form class="search">
-            <input type="text" placeholder="Search..." id="search-bar">
+            <input type="text" placeholder="Search by recipe name" id="search-bar">
```

Comment:
I like this improvement, it gives directions to the user passively

Reply:
I agree! I really like this, great job Jenn!